server/backend/tests/crud/test_crud_application.py

```diff
@@ -1,14 +1,20 @@
+import datetime
 import logging
 import os
+from typing import Any, Dict, Union
 
-import api.app.models.model as models
 import api.app.schemas as schemas
+import sqlalchemy
 from api.app.crud import crud_application as crud_application
 
 LOGGER = logging.getLogger(__name__)
 
-def test_getFamApplications(dbSession_famApplication_withdata, applicationData1):
-    db = dbSession_famApplication_withdata
+
+def test_getFamApplications(
+    dbSession_famApplication: sqlalchemy.orm.session.Session,
```

Comment:
Probably can just use "import" as: ( from sqlalchemy.orm import session )

Reply:
> Probably can just use "import" as: ( from sqlalchemy.orm import session )

True... I was thinking in the context of defining the type returned by the fixture that being explicit would provide more clarity around what is being returned.